generator/src/subcommands/build_crate.rs

```diff
@@ -92,6 +101,8 @@ pub fn invoke(
         cargo.arg("--release");
     }
 
+    let mut rustflags = vec![matches.value_of(RUSTFLAGS).unwrap_or_default().to_owned()];
```

Comment:
Don't this create a vector with an empty string instead of an empty vector, when rustflag is not specified?
Maybe you want something like this?

```suggestion
    let mut rustflags = matches.value_of(RUSTFLAGS).iter().collect::<Vec<_>>();
```

But i also don't really see the need to create a Vec for this instead of just appending to a String as it was before.

Reply:
Thanks for catching this. You are right, as is, there is not really a reason to use a Vec. Reverted back to pushing to a String.

Reply:
> Feel free to rebase and merge.

Sorry about the unneccessary merge commit. Looks like I should have selected "Rebase and merge" in the drop down menu to avoid that. I assumed that github would automatically use fast forward merge if possible.